CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+Version 1.1.2
+------
+* Fix various minor bugs (check dir before creating Rails project, catch stderr from failed git command)
```

Comment:
Since this is external-facing, should we maybe say something like `improve handling of git error messages when creating projects` instead?

Reply:
*&^%, I already merged before seeing your comment 😞